scripts/us_epa/util/superfund_vars.py

```diff
@@ -0,0 +1,144 @@
+_CHEM_MAP = {
```

Comment:
please document these constants, and add copyright header.

also, all the entries in this map seem to have key == value. is there a more programmatic way of dealing with this list?

Reply:
- added documentation from the constants and the copyright header.

- the elements and compounds we have in the map can be handled programatically since the name corresponds to the dcid after some string manipulation (converting to Title Case). But the `_CHEM_MAP` will start becoming more useful when we add the organic compound contaminants where the name in the dataset will be mapped to a different IUPAC name as dcid. The current implementation is just for us to have all contaminants in one place.

Reply:
ok, this sounds good.